tests/test_other.py

```diff
@@ -3595,6 +3595,10 @@ def test_dashM_respect_dashO(self):
     assert len(with_dash_o) == 0
     assert len(without_dash_o) != 0
 
+  def test_null_device_as_input(self):
+    out = Popen([PYTHON, EMXX, '-E', '-dM', '-xc', '/dev/null'], stdout=PIPE).communicate()[0]
```

Comment:
Yes, I think we can just do `"/dev/null" if not WINDOWS else "nul"` in this test, and then it would run on all platforms.

The test suite is run on windows machine on our bots, however some tests might be disabled on windows, so perhaps you saw one of those? @juj, do we not run all `other` tests on windows?